src/assets/theme/customThemeBlocks.ts

```diff
@@ -0,0 +1,27 @@
+import { COLOR_MAP } from 'common/colors';
+
+/**
+ * Used for mega menu and footer, which are the
+ * same component (MenuContent.tsx) with differrent colors and alignment.
+ */
+export const megaMenu = {
+  primaryText: 'black',
+  secondaryText: COLOR_MAP.GREY_4,
+  gray: COLOR_MAP.GREY_4,
+  buttonContent: COLOR_MAP.NEW_BLUE_PURPLE,
+  buttonContentHover: 'white',
+  buttonBorder: COLOR_MAP.GREY_2,
+  aboutUsContentAlignment: 'flex-start',
+  aboutUsTextAlignment: 'left',
+};
```

Comment:
Should we define an interface for this? I think it might help us make sure that theme variants are not missing props